react/QuantitySelector.tsx

```diff
@@ -15,19 +15,23 @@ const MAX_ITEM_QUANTITY = 99999
 const CSS_HANDLES = ['quantitySelectorContainer'] as const
 
 type QuantitySelectorMode = 'default' | 'stepper'
+export type QuantitySelectorStepType = 'unitMultiplier' | 'singleUnit'
 
 interface Props {
   mode?: QuantitySelectorMode
+  quantitySelectorStep?: QuantitySelectorStepType
 }
 
-const QuantitySelector: VFC<Props> = ({ mode = 'default' }) => {
+const QuantitySelector: VFC<Props> = ({ mode = 'default', quantitySelectorStep = 'unitMultiplier' }) => {
   const { item, loading, onQuantityChange } = useItemContext()
   const handles = useCssHandles(CSS_HANDLES)
-
```

Comment:
Bring this line back. Can you lint this file?

Reply:
I bring the line back. How can i lint the file?

Reply:
@germanBonacchi install project dependencies and run `yarn lint`